books/free-programming-books-ko.md

```diff
@@ -49,7 +45,7 @@
 * [BeeJ's Guide to Network Programming - 인터넷 소켓 활용](https://blogofscience.com/Socket_Programming-KLDP.html)
 
 
-<h3 id="cpp">C++</h3>
+### C++
```

Comment:
not a bug

Reply:
Oh, I thought it was an error that didn't fit Mark-down syntax. I put it back to how it was!

Reply:
Resubmit PR: [5868]https://github.com/EbookFoundation/free-programming-books/pull/5868